tck/src/main/java/org/eclipse/microprofile/config/tck/CDITestsFunctions.java

```diff
@@ -0,0 +1,43 @@
+package org.eclipse.microprofile.config.tck;
```

Comment:
In your repo, you only add classes from TCK into your deployment. How do you include classes required by the implementation? This is the sole purpose of the `MicroProfileConfigTestConnector` so far - I needed to add classes from the implementation into deployment archive. It should also be possible to create an arquillian container that contains those classes, but I don't know how to do that. I was using Weld Arquillian container - I would need to add those classes into its classpath.